elasticsearch/utils/set_sg_replicas

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+source "logging"
+
+info "Modifying the Searchguard index replica settings."
+
+config=${ES_CONF}/elasticsearch.yml
+index=$(python -c "import yaml; print yaml.load(open('${config}'))['searchguard']['config_index_name']" | xargs -i sh -c "echo {}")
+
+info "Disabling auto replication"
+${ES_HOME}/plugins/openshift-elasticsearch/sgadmin.sh \
+    -cd ${HOME}/sgconfig \
+    -i $index \
+    -ks /etc/elasticsearch/secret/searchguard.key \
+    -kst JKS \
+    -kspass kspass \
+    -ts /etc/elasticsearch/secret/searchguard.truststore \
+    -tst JKS \
+    -tspass tspass \
+    -nhnv \
+    -icl \
+    -dra ${DEBUG:+-dg}
+
+info "Updating replica count to 0"
+${ES_HOME}/plugins/openshift-elasticsearch/sgadmin.sh \
+    -cd ${HOME}/sgconfig \
+    -i $index \
+    -ks /etc/elasticsearch/secret/searchguard.key \
+    -kst JKS \
+    -kspass kspass \
+    -ts /etc/elasticsearch/secret/searchguard.truststore \
+    -tst JKS \
+    -tspass tspass \
+    -nhnv \
+    -icl \
+    -us 0 ${DEBUG:+-dg}
```

Comment:
Why can't we issue both sgadmin.sh options, `-us 0 -dra` in one sgadmin.sh script invocation?

Reply:
In the code, the will only update one of the operations and then exit. Its required to be two calls at this point. I've created an issue requesting the ability to do this.